automation/src/test/scala/org/broadinstitute/dsde/workbench/leonardo/runtimes/RuntimeDataprocSpec.scala

```diff
@@ -184,44 +184,44 @@ class RuntimeDataprocSpec extends BillingProjectFixtureSpec with ParallelTestExe
         _ = startScriptOutputs.size shouldBe 1
         _ = startScriptOutputs.foreach(o => o.trim shouldBe "This is a start user script")
 
-        // stop the cluster
```

Comment:
Worth discussing, but I think this can be removed entirely. This is essentially verifying google's functionality in terms of what it does with underlying worker nodes when you "stop" a cluster...

We already have stop/start tests

Reply:
if you now where these are being tested ... I would add a link to this comment thread and consider this code deleted

Reply:
Yeah, if we agree it can be removed I agree